kubeflow/trainer/rhai/instrumentation/transformers_checkpoint.py

```
@@ -0,0 +1,1030 @@
+                            "Upload", size
+                        ),
+                    )
+                    uploaded_count += 1
+
+            if uploaded_count == 0:
+                _log("No final model artifacts to upload")
+                return
+
+            _log("Final model upload complete")
+
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)